numba/core/interpreter.py

```diff
@@ -17,6 +17,13 @@
 from numba.core.byteflow import Flow, AdaptDFA, AdaptCFA
 from numba.core.unsafe import eh
 
+class _UNKNOWN_VALUE(object):
```

Comment:
This class needs a docstring.

Am I right that this class has no other use than showing up for debugging?

Reply:
Done in 8602e774b